config/seata-config-etcd3/src/main/java/org/apache/seata/config/etcd3/EtcdConfiguration.java

```diff
@@ -394,7 +394,7 @@ public void onNext(WatchResponse watchResponse) {
                         List<KeyValue> keyValues = getResponse.getKvs();
                         if (CollectionUtils.isNotEmpty(keyValues)) {
                             event.setDataId(dataId).setNewValue(keyValues.get(0).getValue().toString(UTF_8));
-                            listener.onChangeEvent(event);
+                            listener.onProcessEvent(event);
```

Comment:
I'm worried that the user's extended implementation doesn't call onProcessEvent this way.
我担心用户的扩展实现里并不会这样去调用onProcessEvent

Reply:
What I am defining here is the interface standard, and what users rely on is also the interface, which is the basic polymorphic pattern of object-oriented programming. Is there a good method that would allow us to inversely constrain user implementations?

Reply:
> What I am defining here is the interface standard, and what users rely on is also the interface, which is the basic polymorphic pattern of object-oriented programming. Is there a good method that would allow us to inversely constrain user implementations?

Perhaps we can keep the behavior of onChangeEvent consistent with that of onProcessEvent and add a new method to replace the behavior of the original onChangeEvent?
或许我们可以将onChangeEvent的行为保持和onProcessEvent一致,新增一个方法来代替原来的onChangeEvent的行为?

Reply:
The original configuration center had two methods of invocation: onChangeEvent and onProcessEvent. Which do you think should be the standard? The current changes have unified all previous onChangeEvent calls to onProcessEvent. What kind of expectations do you think this has for user behavior? For users to extend on their own, it means they need to understand structurally the presence or absence of cache, as well as the difference between synchronous and asynchronous operations.

Reply:
Because the configuration is mainly used by seata, not the user, if this change can't make the user-side implementation behave the same as our implementation, it will affect the question of whether the configuration is effective in seata or not. So this should be a strong constraint setting, all notifications should go onProcessEvent behavior, otherwise the user's extension will lose a lot of functionality without perception
因为配置主要是由seata使用，并不是用户使用，如果这个改动无法让用户侧的实现跟我们的实现行为一致，这将影响到配置在seata中是否生效的问题。所以这块应该是强约束的设定，所有的通知应该都走onProcessEvent这个行为，否则用户的扩展将在无感知的情况下丧失很多功能